tox.ini

```diff
@@ -1,10 +1,57 @@
 [tox]
-envlist = py310
+envlist = actions_build, local_test
 skipsdist = True
 
 [testenv]
 deps =
     -rrequirements.txt
+    flake8
+    flake8-pie
+    flake8-simplify
+    flake8-bugbear
+    flake8-typing-imports
+    pep8-naming
+    wemake-python-styleguide
+    flake8-encodings
+    flake8-comprehensions
+    flake8-implicit-str-concat
+    flake8-return
+    flake8-rst-docstrings
+whitelist_externals =
+    bash
+
+[flake8]
+ignore = D203
+# Increase default E501 restriction (79 chars) with 100
+max-line-length = 120
```

Comment:
@dborowiecki Is it an inconsistency? (in comment we have 100 but value for this option=120)

Reply:
Good catch, fixed